docs/RELEASE_NOTES.md

```diff
@@ -26,24 +26,27 @@
 
 7. Add Migration guide section which specifies explicitly if there are breaking changes and how to tackle them.
 -->
-### 4.0.1 - Mar 19, 2021
+
+    
+### 4.1.0 - ???
```

Comment:
This is a breaking change btw so we might need to increase the version to 5.0.0, but I feel this branch should not be merged to master until we finish the related work in this sprint's iteration. Otherwise 2 breaking change would not be nice for our library users. Could you please base this branch to a branch called `development_5.0.0` ( you would name it better) ?

Reply:
46e939eb0 and I have changed the base branch of this pr to 'release_5_0_0'